allennlp/data/vocabulary.py

```diff
@@ -3,13 +3,18 @@
 
 import codecs
 import logging
-import tqdm
+import os
 
 from allennlp.common.util import namespace_match
 
+import tqdm
+
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 DEFAULT_NON_PADDED_NAMESPACES = ("*tags", "*labels")
+DEFAULT_PADDING_TOKEN = "@@PADDING@@"
+DEFAULT_OOV_TOKEN = "@@UNKNOWN@@"
```

Comment:
Is there a need for these to be in the global scope? Can they be class variables?